.github/workflows/developing.yml

```diff
@@ -22,6 +22,9 @@ jobs:
   check:
     runs-on: ubuntu-latest
     steps:
+      - name: Maximize build space
+        uses: easimon/maximize-build-space@master
```

Comment:
`easimon/maximize-build-space` supports to set `remove-dotnet: 'true'`, we can replace the original `rm /usr/share/dotnet` with this action in every job.

Reply:
Also, use `easimon/maximize-build-space@v4` instead to avoid unexpected upstream changes.